### PR TITLE
Add PR Details

### PR DESCRIPTION
**Pull request by kingthorin**
Update CONTRIBUTING.md to include handling details for Pull Requests.

### CODE REVIEW

**Comment by psiinon** [comment]
Looks good

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,3 +72,27 @@ Add-ons are a great way to extend ZAP and can be ideal for student projects - ma
...
+* Descriptive commits (subject and message):
+ * For example: https://github.com/spring-projects/spring-framework/blob/master/CONTRIBUTING.md#format-commit-messages
+* Discussion about the changes:
+ * Should be done in/on the PR or via the Dev Group and a link to that Dev Group thread added to the PR comments. (i.e.: Shared information is important, if something happens via IRC or private email please ensure a summary makes it to the PR.)
+ * Discussion will be kept in the pull request unless off topic.
+* No merge commits. Please, rebase.
+* Rebase if the branch has conflicts.
+* How much time will a pull request be left open?
+ * This isn�t static, one or more members of the ZAP Team will reach out (using @ mentions in PR comments) once or twice in order to get things back on track. If no input is received after a month or two then the PR will be closed. Total stale time will likely be 2 to 3 months.
+  * Close with a message such as: "The pull request was closed because of lack of activity (as per CONTRIBUTING guidelines)". Labeled as "Abandon".
```
Comment (thc202) [inline CONTRIBUTING.md]: Could the label be "pr:stale" instead? It's more descriptive IMHO.

Comment (kingthorin) [inline CONTRIBUTING.md]: Sure I'm good with that. I've updated the two label sets [1,2], will update the PR tonight.

[1] https://github.com/zaproxy/zap-extensions/labels
[2] https://github.com/zaproxy/zaproxy/labels

Comment (thc202) [inline CONTRIBUTING.md]: OK, thanks!

**Comment by kingthorin** [comment]
Test @zaproxy/team-zaproxy

**Comment by thc202** [comment]
Looks good, although the last commit should be fixed up.

**Comment by kingthorin** [comment]
No problem

**Comment by thc202** [comment]
Looks good to me, thanks!

**Pull request merged**